docroot/themes/humsci/humsci_basic/templates/html.html.twig

```diff
@@ -62,6 +62,10 @@
     <meta name="msapplication-square150x150logo" content="https://www-media.stanford.edu/assets/favicon/mstile-150x150.png" />
     <meta name="msapplication-square310x310logo" content="https://www-media.stanford.edu/assets/favicon/mstile-310x310.png" />
 
+    {# Google Fonts Source Sans 3 and Source Serif 4. #}
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
Is there a reason for not adding these 2 `<link>` in the preprocess, along with the stylesheets? It would be better to add everything in the same place.